waku/protocol/v2/waku_filter.nim

```diff
@@ -0,0 +1,96 @@
+import chronos, chronicles
```

Comment:
How is this file different from the existing `v2/filter.nim`? Should that be removed? It is used in other PR and interface seems different.